pkg/cmd/release/view/view_test.go

```diff
@@ -244,6 +247,7 @@ func Test_viewRun(t *testing.T) {
 			shared.StubFetchRelease(t, fakeHTTP, "OWNER", "REPO", tt.opts.TagName, fmt.Sprintf(`{
```

Comment:
Ideally, I'd love to see a test case with `--json <all-fields>` here, but since we don't have the test already I think we can skip it for this PR.